core/gui/src/app/dashboard/user/component/user-dataset/user-dataset-explorer/user-dataset-version-creator/user-dataset-version-creator.component.ts

```diff
@@ -87,10 +88,10 @@ export class UserDatasetVersionCreatorComponent implements OnInit {
           {
             key: "versionName",
```

Comment:
Do you plan to update the key here? Still calling it `name` and `versionName` might cause confusions.

Reply:
good catch. I made them consistent